packages/pn-personagiuridica-webapp/src/App.tsx

```diff
@@ -200,6 +203,15 @@ const ActualApp = () => {
     });
   }
 
+  if (IS_B2B_ENABLED) {
```

Comment:
This is the only condition to show/hide the menu? On pg we have four user types:
- admin
- admin with groups
- operator
- operator with groups

I think that operator and operator with groups, shouldn't access this route. I'm not sure about the admin with groups

Reply:
As decided in SU, this page must be shown to all users